### PR TITLE
Add suggestions originally from @martinda

### PR DESCRIPTION
**Pull request by lloydchang**
References:
1. Comments for git commit https://github.com/jenkinsci/jep/commit/b7fe523450c1c34a57c61923695d3b9049280dff#comments
2. Pull Request #144

Co-Authored-By: Martin d'Anjou <martin.danjou14@gmail.com>
Cc @martinda @jeffpearce @oleg-nenashev

### CODE REVIEW

File: jep/8/README.adoc [hunk 1]
```diff
@@ -144,16 +141,14 @@ and then seek reimbursement for the expenses. Pre-approval is still required in
...
 
 This means:
 
 * Approval by GSoC org admins does not guarantee the payments
-* Jenkins project, GSoC org admins or SPI cannot guarantee ETA of any payment or, in the worst case,
+* Jenkins project, GSoC org admins or SPI cannot guarantee the timing of any payment or, in the worst case,
 the payment itself
-* Jenkins Project or SPI cannot issue guarantee letters for
-visa applications or other or other travel documents.
+* Jenkins Project or SPI cannot issue guarantee letters for visa applications or other or other travel documents.
```
Comment (oleg-nenashev) [inline jep/8/README.adoc]: "Or other" is duplicated (not a new issu tho)

File: jep/8/README.adoc [hunk 2]
```diff
@@ -109,8 +107,7 @@ open-source conferences like FOSDEM, etc.
 ** Project expenses should happen within the GSoC timeframe between start of Community Bonding and end of the last coding phase
 ** GSoC budget may not be used to cover generic equipment costs: laptop, IDEs, internet, etc.
 ** GSoC budget may not be used to cover software license costs.
-The most of the projects are expected to be doable with only open-source or freeware software,
-otherwise it is up to the students and mentors to get the setup.
+Should a project require access to non-free resources (software or hardware), the project mentors and student are expected to obtain the resources at no cost to the GSoC budget.
```
Comment (oleg-nenashev) [inline jep/8/README.adoc]: I propose to discuss it. Some requests may be viable (e.g. SaaS), so I would permit discussing options if such question arises. Org admins may help to reach out to potential sponsors

File: jep/8/README.adoc (hunk 1 above)
Comment (bitwiseman) [reply jep/8/README.adoc]:
```suggestion
* Jenkins Project or SPI cannot issue guarantee letters for visa applications or other travel documents.
```

File: jep/8/README.adoc (hunk 2 above)
Comment (lloydchang) [reply jep/8/README.adoc]:
```suggestion
Should a project require access to non-free resources (software or hardware), the project mentors and student are expected to discuss options with org admins. Org admins may help to reach out to potential sponsors.
```

**Comment by oleg-nenashev** [comment]
Thanks @martinda @lloydchang and @bitwiseman !

**Pull request merged**